src/Microsoft.AspNetCore.SignalR.Protocols.Json/Protocol/JsonHubProtocol.cs

```diff
@@ -544,6 +592,17 @@ private HubMessage BindCancelInvocationMessage(string invocationId)
             return new CancelInvocationMessage(invocationId);
         }
 
+        private HubMessage BindStreamCompleteMessage(string streamId, string error)
```

Comment:
Should you add a check on `error` in this method?

Reply:
It's okay because error isn't a required property. The `error` field may be null, which indicates that the stream completed without error.

Reply:
Yeah, so you should add a check to make sure that the error is in fact null to be sure nothing bad happened 😄

Reply:
There's nothing to check here. Error could be null or it could be non-null, both are valid.